iris_grib/__init__.py

```diff
@@ -1028,19 +1028,6 @@ def save_grib2(cube, target, append=False, **kwargs):
     save_messages(messages, target, append=append)
 
 
-def as_pairs(cube):
-    """
-    .. deprecated:: 1.10
```

Comment:
Ooh, this is a little more tricky: Iris v1.10 is the current version so can we say this has observed its deprecation period?

Answers on a postcard :postbox: